app/src/main/res/layout/content_main.xml

```diff
@@ -56,18 +61,27 @@
         android:id="@+id/txtActivity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginTop="92dp"
         android:ems="10"
+        android:hint="@string/activity_hint"
         android:inputType="textPersonName"
-        android:text="Activity..."
-        tools:layout_editor_absoluteX="99dp"
-        tools:layout_editor_absoluteY="190dp" />
+        android:text="@string/activity"

```

Comment:
Created string in strings.xml and replaced text with @string. This removed this warning in android studio about using string values.